src/styles/style.css

```diff
@@ -462,10 +465,75 @@
 }
 
 /* -----------------END DETAIL----------------- */
-/* -----------------HACKS----------------- */
 
-.none {
-  display: none;
+/* -----------------RESPONSIVENESS----------------- */
+
+@media screen and (max-width: 70em) {
+  #detailBook {
+    position: initial;
+    box-shadow: none;
+    height: auto;
+  }
+
+  .detailCover {
+    padding: 5rem 1rem 1rem 1rem;
+    width: 100%;
+  }
+
+  .detailInformation {
+    width: 100%;
+    padding: 7rem 7rem 5rem 7rem;
+  }
+
+  .detailInformation h1 {
+    font-size: calc(100vw*0.09);
+  }
+
+  .detailInformation p:nth-of-type(1) {
+    font-size: calc(100vw*0.04);
+  }
+
+  .detailInformation p:nth-of-type(2) {
+    font-size: 2.3em;
+  }
+
+  .detailInformation p:nth-of-type(4) {
+    font-size: 2.3em;
+    margin-top: 3em;
+  }
+
+  .detailInformation button {
+    font-size: 2.3em;
+  }
+
+  #detailRecommendation section p {
+    font-size: 2.3em;
+  }
+
+  #detailRecommendation {
+    position: initial;
+  }
+
+  .detailCover img {
+    width: 100%;
+    height:100%;

```

Comment:
to size an image you can ether use with OR height. You don't have to style both. (also missing a space after the ":")